data_diff/tracking.py

```diff
@@ -85,12 +85,14 @@ def create_start_event_json(diff_options: Dict[str, Any]):
 def create_end_event_json(
     is_success: bool,
     runtime_seconds: float,
-    db1: str,
-    db2: str,
+    data_source_1_type: str,
```

Comment:
I changed these just because I found the names confusing/misleading

Reply:
non-blocking: I'll need to update downstream reporting

Reply:
this just impacts the python code, the actual event is not affected

Reply:
oh I see, still "data_source_1_type" 👍